packages/nimble-tokens/data/fonts.json

```diff
@@ -868,21 +870,21 @@
       },
       "tokens": [
         {
-          "id": "9a9732b5-22c4-41e9-baa5-54f758d152ca",
+          "id": "Error_ColorUI-family",
```

Comment:
Just to double-check these changes were made by going through the DSP plugin editor and using the edit mode and interactive UI for making changes?

Are there any changes we should make to Contributing for future users going though the manual update workflow? https://github.com/ni/nimble/blob/main/packages/nimble-tokens/CONTRIBUTING.md#editing-tokens

Reply:
I thought the current instructions were pretty good. I followed the instructions for "token metadata changes" since that's what has to be manually done.

Reply:
Sounds good 👍